services/user.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+from django.contrib.auth import get_user_model
```

Comment:
But here you use get_user_model, isn't it strange? 👀

Reply:
Yes, that's because I had python 3.9. Now I have updated it)